meshnesh/react-consume-api
# Ft authorized routes

# What does this PR Do ?
* Create Component responsible for creating events by a person whose email has been verified.

## How should this be manually tested?

* Clone the code git clone https://github.com/meshnesh/react-consume-api.git
* Move to the right foldercd react-consume-api
* Run the project locally npm run dev
* Open the application on [local host port](http://localhost:8080/)
* Create and account them verify the email account from the email.
* Navigate to manage events on the navigation tab, click on the floating action button to be redirected to the adding event component. Create a new event.

## Relevant pivotal tacker stories
* [#155327195](https://www.pivotaltracker.com/story/show/155327195)
